### PR TITLE
Add System.Math

### PR DESCRIPTION
**Pull request by shargon**
Close https://github.com/neo-project/neo-devpack-dotnet/issues/587

### CODE REVIEW

File: src/Neo.Compiler.CSharp/MethodConvert.cs [hunk 1]
```diff
@@ -4109,6 +4109,44 @@ private bool TryProcessSystemMethods(SemanticModel model, IMethodSymbol symbol,
                         PrepareArgumentsForMethod(model, symbol, arguments);
                     Call(NativeContract.StdLib.Hash, "atoi", 1, true);
                     return true;
+                case "System.Math.Pow(double, double)":
```
Comment (erikzhang) [inline src/Neo.Compiler.CSharp/MethodConvert.cs:4112]: We don't have `double` in contracts.

Comment (shargon) [reply src/Neo.Compiler.CSharp/MethodConvert.cs:4112]: We can use a casted integer

Comment (erikzhang) [reply src/Neo.Compiler.CSharp/MethodConvert.cs:4112]: We should use `BigInteger.Pow`.

Comment (shargon) [reply src/Neo.Compiler.CSharp/MethodConvert.cs]: Done

File: src/Neo.Compiler.CSharp/MethodConvert.cs [hunk 2]
```diff
@@ -4109,6 +4109,38 @@ private bool TryProcessSystemMethods(SemanticModel model, IMethodSymbol symbol,
                         PrepareArgumentsForMethod(model, symbol, arguments);
                     Call(NativeContract.StdLib.Hash, "atoi", 1, true);
                     return true;
+                case "System.Math.Sign(byte)":
```
Comment (nicolegys) [inline src/Neo.Compiler.CSharp/MethodConvert.cs:4112]: shouldn't it be `sbyte`?

Comment (shargon) [reply src/Neo.Compiler.CSharp/MethodConvert.cs]: Fixed

**Review by nicolegys — APPROVED** [review]
Test passed.

**Review by erikzhang — APPROVED** [review]

**Pull request merged**